var/spack/repos/builtin/packages/allinea-forge/package.py

```diff
@@ -16,6 +25,7 @@ class AllineaForge(Package):
 
     version('6.0', 'c85fec6d01680b5b46fea80111186244')
     version('7.0', '4c2f5b2ff83d494854df74e6df4be7be')
+    version('16.04', 'd8396b046e9b7f4241d69466f6155790')
```

Comment:
This is messed up. @pramodk added versions here. Correct would be `latest`, and at most variants for the distro? IMO, no variants needed actually.